packages/common/client-utils/src/test/types/tsconfig.json

```diff
@@ -1,19 +1,21 @@
 {
-    "extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": [
```

Comment:
Some indentation issues in this file, looks like

Reply:
The root cause of this is fixed in https://github.com/microsoft/FluidFramework/pull/18207.